src/TemplateLibrary/index.js

```diff
@@ -73,6 +84,7 @@ class TemplateLibrary extends React.PureComponent {
 
   componentDidMount() {
     // this.fetchTemplates();
```

Comment:
let's also remove this and the corresponding instance method if we are not using it

Reply:
Right, this makes sense. I had the `CustomLoader` and this functionality in place for future implementations of that functionality. It makes more sense to not push those to master until they are actually developed.

Reply:
You can checkout a new branch locally and save them there so that you have the code in that branch when you need it :)